Add React Native V6 Changes (#11320)
getsentry/sentry-docs: docs/platforms/react-native/tracing/instrumentation/custom-navigation.mdx

sidebar_order: 70
---

> Sentry's React Native SDK package ships with instrumentation for [React Navigation](/platforms/react-native/tracing/instrumentation/react-navigation/) and [React Native Navigation](/platforms/react-native/tracing/instrumentation/react-native-navigation/) and [Expo Router](/platforms/react-native/tracing/instrumentation/expo-router/). This allows you to see the performance of your navigation transitions and the errors that occur during them.

Reviewer: ```suggestion
Sentry's React Native SDK package ships with instrumentation for [React Navigation](/platforms/react-native/tracing/instrumentation/react-navigation/), [React Native Navigation](/platforms/react-native/tracing/instrumentation/react-native-navigation/), and [Expo Router](/platforms/react-native/tracing/instrumentation/expo-router/). This allows you to see the performance of your navigation transitions and the errors that may occur.
```
